bayesian/utility/v2/ca_response_builder.py

```diff
@@ -20,7 +20,7 @@
 
 from bayesian.utility.db_gateway import GraphAnalyses
 from f8a_utils.user_token_utils import UserStatus
-from bayesian.utils import version_info_tuple, convert_version_to_proper_semantic
+from f8a_version_comparator.comparable_version import ComparableVersion
```

Comment:
You need to explicitly add f8a_version_comparator in requirments.in.

Reply:
This dependency was existing in requirements.txt, I am just using it here. Anyways it should be present in requirements.in also. Look like there was pending issue. I will fix it.

Reply:
requirements.txt contains both direct and trans dependencies., what if we stop using fa8_utils in future?

Reply:
@arajkumar in that case, the unit test will fail and will get to know about a missing dep

Reply:
We are following this as a convention across the repo. requirement.in must be a subset of requirements.txt